.github/workflows/e2e-test-playwright.yml

```
@@ -0,0 +1,51 @@
+name: Playwright Tests
+
+on:
+    pull_request:
+        branches: [master]
+
+jobs:
+    e2e-js-playwright:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)